qemu/tests/cfg/hotplug_virtio_mem.cfg

```diff
@@ -0,0 +1,37 @@
+- hotplug_virtio_mem:
+    only Linux
+    only Host_RHEL.m9
+    only RHEL.9
+    no s390x
+    type = hotplug_virtio_mem
+    virt_test_type = qemu
+    login_timeout = 240
+    threshold = 0.025
+    aarch64:
+        smp = 8
+        vcpu_maxcpus = ${smp}
+    slots_mem = 20
+    maxmem_mem = 80G
+    mem = 4096
+    mem_devs = 'mem0 mem1'
+    guest_numa_nodes = 'node0 node1'
+    numa_memdev_node0 = mem-mem0
+    numa_memdev_node1 = mem-mem1
+    numa_cpus_node0 = "0-3"
+    numa_cpus_node1 = "4-7"
```

Comment:
```suggestion
```

Reply:
@PaulYuuu why are you suggesting to delete this block? The polarion case uses this option:

```bash
-numa node,memdev=mem0,cpus=0-3 \
-numa node,memdev=mem1,cpus=4-7 \
```

Reply:
Since you plan to delete `smp = 8`, so this is also not necessary. If the case is running with 4 CPU, these 2 lines will be a problem.

Reply:
Changed now, thanks!